dockerfiles/Dockerfile.windows

```diff
@@ -38,6 +38,11 @@ RUN Start-Process /local/vc_redist.x64.exe -ArgumentList '/install', '/quiet', '
     Copy-Item -Path /Windows/System32/vccorlib140.dll -Destination /fluent-bit/bin/; `
     Copy-Item -Path /Windows/System32/vcruntime140.dll -Destination /fluent-bit/bin/;
 
+# Install ZLIB: https://github.com/horta/zlib.install
+ADD https://github.com/alt-dima/zlib.install/raw/master/install.bat /local/install_zlib.bat
```

Comment:
We probably should not depend on `master` of another repo we have no control over - potentially this is updated and breaks the build best case (worst case it's a malware attack vector). We should either put the commands inline in the Dockerfile or move the script into this repo as part of the PR - then it can be versioned alongside.